src/components/editor/Controls.tsx

```diff
@@ -12,37 +13,70 @@ const Label = ({ text, children }: {
 	</label>
 }
 
-export default function Controls({ exportHandler, exporting }: {
-	exportHandler: (
-		data: {
-			fps: number,
-			size: number,
-			speed: number,
-			bounce: boolean,
-			loop_gif: boolean
-		}) => void,
-	exporting: boolean
+const getEstimatedFileSize = (
+	fps: number,
+	width: number,
+	height: number,
+	quality: number,
+	durationInFrames: number
+) => {
+	const durationInSeconds = durationInFrames / CONSTANTS.previewFps;
```

Comment:
I'm not sure but does `durationInSeconds` affect the output size?
When we increase the duration ( by decreasing the speed ) we are doubling the timestamps, but the total number of frames in the gif should stay the same so the size should not be affected by the duration.
Thoughts @clearlysid ?

Reply:
- duration here = total no. of frames that need to be rendered.
- file size doesn't get affected by "speed", but it does by "duration".
- a 20 second GIF will be 2x as large as a 10 second GIF right?

Reply:
Yes. Agree with the 3 statements.

So isn't `durationInFrames` the duration we want? Can we not use it directly instead of (inaccurately) converting it to seconds and multiplying by fps ? i.e `totalFrames = durationInFrames`.

> a 20 second GIF will be 2x as large as a 10 second GIF right?
Yup. But will a 20 second GIF at 15fps be the same size as a 10 second GIF at 30fps ?